spec/serializers/api/internal/v1/defendant_serializer_spec.rb

```diff
@@ -1,41 +1,81 @@
 # frozen_string_literal: true
 
 RSpec.describe Api::Internal::V1::DefendantSerializer do
-  subject { described_class.new(defendant).serializable_hash }
-
-  let(:defendant) do
-    instance_double("Defendant",
-                    id: "UUID",
-                    name: "John Doe",
-                    date_of_birth: "2012-12-12",
-                    national_insurance_number: "XW858621B",
-                    arrest_summons_number: "MG25A11223344",
-                    maat_reference: "123123",
-                    representation_order_date: "2020-12-12",
-                    offence_ids: %w[55555],
-                    defence_organisation_id: "88888",
-                    prosecution_case_id: "5edd67eb-9d8c-44f2-a57e-c8d026defaa4",
-                    post_hearing_custody_statuses: %w[A])
-  end
+    let(:defendant) do
+      defendant_summary_data = JSON.parse(file_fixture("defendant_summary/all_fields.json").read)
+      defendant_data = JSON.parse(file_fixture("defendant/all_fields.json").read)
+      Defendant.new(body: defendant_summary_data, details: [defendant_data], prosecution_case_id: "123")
```

Comment:
These 3 lines make all of this much clearer, thank you!

Reply:
my pleasure!